core/lib/lantern/simulation/SimulationTimingMap.js

```diff
@@ -4,7 +4,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import {BaseNode} from '../BaseNode.js';
+import * as Lantern from '../lantern.js';
```

Comment:
nit: the old way may still be valid since we're in the `lantern/` directory already. Also would this cause any sort of circular dependency situation?